source/server/server.cc

```diff
@@ -855,6 +862,16 @@ void InstanceImpl::loadServerFlags(const absl::optional<std::string>& flags_path
   }
 }
 
+void InstanceImpl::configureBackgroundMemoryRelease() {
```

Comment:
Would instead suggest pushing the memory configuration / release rate logic into https://github.com/envoyproxy/envoy/blob/main/source/common/memory/stats.h or similar file under `memory/` to encap which allocator, etc. logic into one place vs bring it out here.

Reply:
yeap, i was not sure where to put it, one assumption was that we want to configure this as early as possible. But logically putting into memory makes the most sense.